mender/values.yaml

```diff
@@ -546,11 +546,11 @@ generate_delta_worker:
       whenScaled: "Delete"
   resources:
     limits:
-      cpu: 100m
-      memory: 1024Mi
+      cpu: 2
+      memory: 4096Mi
     requests:
-      cpu: 100m
-      memory: 128Mi
+      cpu: 1
+      memory: 2048Mi
```

Comment:
```suggestion
resources:
limits:
      cpu: 2
      # Warning: if your file size exceeds this memory limit,
      # the delta generation could fail.
      # Please tune it with:
      # <your maximum file size> + <your maximum file size> * 20%
      memory: 4096Mi
requests:
      cpu: 1
      memory: 2048Mi
```

Reply:
yeah good one. there is only one thing I have to do, read xdelta3 sources, because I bet we have to mention at least one window size (sine you can get crazy with the parameters to the generator, I have to check it allocates memory vs the values given)

Reply:
fixed/added. thanks.